### PR TITLE
Quote bash variable expansion.

### PR DESCRIPTION
**Pull request by yig**
Variable expansion should be quoted.

### CODE REVIEW

File: letsencrypt-auto [hunk 1]
```diff
@@ -114,24 +114,24 @@ DeterminePythonVersion() {
 # later steps, causing "ImportError: cannot import name unpack_url"
 if [ ! -d $VENV_PATH ]
 then
-  if [ ! -f $BOOTSTRAP/debian.sh ] ; then
+  if [ ! -f "${BOOTSTRAP}"//debian.sh ] ; then
```
Comment (convissor) [inline letsencrypt-auto]: Was adding the second slash a typo?

Comment (yig) [inline letsencrypt-auto]: Yes :-p

Comment (convissor) [inline letsencrypt-auto]: Happens.  Fix this and the other typos and push it up to your remote.

**Comment by yig** [comment]
Already done.

**Comment by convissor** [comment]
Yep.  Noticed Github is saying conflicts have arisen since you first authored the patch.

**Comment by yig** [comment]
I fixed the conflicts and pushed a new change. The failing test is due to a DNS timeout. It doesn't look related to my changes. I'm not sure how to re-run it.

**Comment by bmw** [comment]
Kicked Travis.

[Log](https://gist.github.com/3efa3e1fbc57977c462c) of letsencrypt/boulder#1237.

**Comment by erikrose** [comment]
Fixed most of these in #1665. Ported the remainder to the latest le-auto in #2464. Thanks for pointing this out!

**Pull request closed without merging**